lib/xport/formatters/axlsx.rb

```diff
@@ -13,6 +13,36 @@ def to_xlsx(&block)
       to_file(formatter, &block)
     end
 
+    def to_pgxlsx
+      formatter = Xport::Axlsx::Formatter.new(self)
```

Comment:
Would be really great if this could be used with any formatter - axlsx, xlsxtream, csv, etc.

```
module PostgresCopy
    def write_body(formatter, worksheet)
       ....
    end
end
```

```
module PostgresCSV
    def to_csv
       ....
    end
end
```

and then we could use

```
class UserExport < Xport::Export
  include Xport::CSV
  include Xport::Axlsx
  include Xport::PostgresCopy # can be used for csv and excel
  include Xport::PostgresCSV # even faster csv (optional)
```

Also test would be very nice addition.